backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/transformer/NormReferenceTransformer.java

```diff
@@ -86,9 +87,11 @@ public static List<NormReferenceDTO> transformToDTO(NormReference normReference)
                   NormReferenceDTO.builder()
                       .id(singleNorm.id())
                       .normAbbreviation(normAbbreviationDTO)
-                      .normAbbreviationRawValue(normAbbreviationRawValue)
-                      .singleNorm(singleNorm.singleNorm())
+                      .normAbbreviationRawValue(
+                          StringUtils.normalizeSpace(normAbbreviationRawValue))
+                      .singleNorm(StringUtils.normalizeSpace(singleNorm.singleNorm()))
                       .dateOfVersion(singleNorm.dateOfVersion())
+                      // dateOfRelevance should LocalDate?
```

Comment:
What does this comment mean?

Reply:
Just a thought from my side, when going over all the data. But I will remove the comment for now, as it has nothing to do with this topic.